### PR TITLE
Preserve directives when adding import statement

### PR DESCRIPTION
**Pull request by bouk**
@lemonmade

### CODE REVIEW

**Comment by bouk** [comment]
This makes it so we can do global-assignment-to-default-export before global-reference-to-import and preserve the expose directive

File: packages/shopify-codemod/transforms/utils.js [hunk 1]
```diff
@@ -11,3 +11,14 @@ export function findLastMember(node) {
   }
   return node;
 }
+
+export function insertAfterDirectives(body, newNode) {
+  let i = 0;
+  for (;i < body.length; i++) {
```
Comment (lemonmade) [inline packages/shopify-codemod/transforms/utils.js]: Ugh, hate this way of expressing this. Can we at least move the `let i = 0` in as the first condition?

Comment (bouk) [inline packages/shopify-codemod/transforms/utils.js]: No, because i is block scoped to for for loop otherwise.

Comment (lemonmade) [inline packages/shopify-codemod/transforms/utils.js]: As shit, yeah. Gross :( could do it with a weird `findIndex` but whatever, probably better this way.

**Comment by lemonmade** [comment]
:+1:

**Pull request merged**